src/components/Tooltip/Tooltip.tsx

```diff
@@ -36,6 +36,8 @@ export const TooltipTrigger = forwardRef<
   HTMLProps<HTMLElement> & { asChild?: boolean }
 >(function TooltipTrigger({ children, asChild = false, ...props }, propRef) {
   const context = useTooltipContext()
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is the second instance where I don't have a good solution to factor-out `any`.

Reply:
Super clever ❤️